Source/WebCore/platform/graphics/cg/IOSurfacePool.h

```diff
@@ -78,7 +78,11 @@ class IOSurfacePool : public ThreadSafeRefCounted<IOSurfacePool> {
     typedef HashMap<IntSize, CachedSurfaceQueue> CachedSurfaceMap;
     typedef HashMap<IOSurface*, CachedSurfaceDetails> CachedSurfaceDetailsMap;
 
+#if PLATFORM(MAC)
+    static constexpr size_t defaultMaximumBytesCached { 1024 * 1024 * 256 };
```

Comment:
We must have a BYTES_PER_MEGABYTE somewhere.

Reply:
Hm... I don't find it anywhere.

Reply:
We have `MB` in StdLibExtras.h, so you can use `256 * MB`.

Reply:
oh neat.